src/redux/detail_slice.js

```diff
@@ -0,0 +1,49 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const API_URL = 'http://127.0.0.1:3000';
+
+const initialState = {
+  itemDetail: null,
+  status: 'idle',
+  error: null,
+};
+
+export const fetchItemDetail = createAsyncThunk(
+  'detail/fetchItemDetail',
+  async (itemId) => {
+    try {
+      const response = await axios.get(`${API_URL}/items/${itemId}`);
```

Comment:
```suggestion
      const response = await axios.get(`${API_URL}/units/${unitId}`);
```

Reply:
Kindly, note that the updated line is using the right API end point for Unit details.